.github/workflows/clang-format-pull-request.yml

```diff
@@ -19,8 +19,8 @@ jobs:
     - name: Applying clang-format for changed files
       run: |
         URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${{ github.event.pull_request.number }}/files"
-        FILES=$(curl -s --request GET --get $URL |  jq -r '.[] | .filename' | grep ^include | grep -v nanorange\.hpp\$)
-        echo $FILES | xargs -n1 -t clang-format-6.0 --style=file -i
+        FILES=$(curl -s --request GET --get $URL |  jq -r '.[] | .filename' | grep ^include || true | grep -v nanorange\.hpp\$ || true)
```

Comment:
Could you please explain the purpose of `|| true`? Especially before `grep -v nanorange`.
As I understand the current set of commands will not exclude nanorange files from the list if they are presented.

Reply:
Reason for adding || true is exit code of grep. It is non zero when zero items matched and github action stops execution marking step as failed.

Reply:
It sounds reasonable for the `|| true` in the end of the line, but `|| true` in the middle seems breaking the behavior.

Reply:
Looks like you are right. Will check this and push fix